packages/dx-react-grid/src/plugins/local-paging.jsx

```diff
@@ -11,6 +11,13 @@ const rowsWithHeadersComputed = ({ rows, pageSize, getRowLevelKey }) =>
 const totalCountComputed = ({ rows }) => rowCount(rows);
 const paginatedRowsComputed = ({ rows, pageSize, currentPage }) =>
   paginatedRows(rows, pageSize, currentPage);
+const gettersChangesComputed = (returnedValue, getters, actions) => {
```

Comment:
Why do you use this several times? Seems like it should be applied only to one getter. Rework it.